src/Files.App/Data/Items/AppLanguageItem.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) 2023 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
+namespace Files.App.Data.Items
+{
```

Comment:
```suggestion
// Licensed under the MIT License. See the LICENSE.

using System.Globalization;

namespace Files.App.Data.Items
{
```

This will prevent the build error

Reply:
Done